src/Types/Int.purs

```diff
@@ -3,21 +3,38 @@ module Types.Int
   ( Int
   , newPositive
   , newNegative
+  , fromBigInt
   ) where
 
+import Prelude
+
 import Prim as Prim
-import Prelude as Prelude
 import Serialization.Types (BigNum)
 import Data.Function (on)
+import Data.BigInt as BigInt
+import Partial.Unsafe (unsafePartial)
+import Data.Maybe (Maybe, fromJust)
+import Control.Alternative ((<|>))
+import Serialization.BigNum (bigNumFromBigInt)
 
 foreign import data Int :: Prim.Type
 
 foreign import newPositive :: BigNum -> Int
 foreign import newNegative :: BigNum -> Int
 foreign import _intToStr :: Int -> Prim.String
 
-instance Prelude.Eq Int where
-  eq = Prelude.eq `on` _intToStr
+instance Eq Int where
+  eq = eq `on` _intToStr
+
+instance Ord Int where
+  compare = compare `on` \number ->
```

Comment:
Just a stylistic thing: I think it would be nicer point-free

Reply:
Type inference breaks when using `unsafePartial` with composition.

```
  Could not match constrained type

    Partial => t0

  with type

    BigInt
```